SETUP.md

```diff
@@ -693,7 +693,7 @@ dashboard.example.com_postfix_1       /sbin/my_init                    Up
 - In the Host machine, a cronjob needs to be configured as below.
 
     ```console
-    sudo echo "0 * * * * /bin/bash -l -c '/opt/docker/dashboard.example.com/apiserver/version_check.sh'" >> /etc/crontab
```

Comment:
You're using tabs, and it's not clear to anyone copying pasting.

Also, you're using `training.mcci.io`; this is not allowed. (We don't use real URLs in examples.)  Please revert to `dashboard.example.com` as `example.com` is guaranteed never to be in use.

If tabs are required, you must use `printf`, and be more careful about quotes, e.g.:

```bash
sudo printf '0\t*\t*\t*\t/bin/bash -l -c /opt/docker/dashboard.example.com/apiserver/version_check.sh\n' '>>' /etc/crontab
```

Note that I avoided the nested quotes as they are not neceesary, and they are extremely troublesome; and I changed to single quotes on the outer quote. And that I quoted the I/O redirection, because `sudo` takes effect after redirect and the `>>` won't work. Not sure why there was a sudo here in the first place, if the `>>` even works as it was given. I assume the `sudo` is actually required and so quoting the `>>` so it will be parsed by the `su` shell. But this must be tested.

Reply:
Okay Terry, we will change this